keyboards/rose75/config.h

```diff
@@ -0,0 +1,14 @@
+// Copyright 2023 Matthijs Muller (@Smollchungus)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+// RGB indicators
+
+#define WS2812_PIO_USE_PPIO1
+#define RGB_MATRIX_LED_COUNT 2
+#define RGB_DISABLE_WHEN_USB_SUSPENDED
+#define RGB_MATRIX_MAXIMUM_BRIGHTNESS 210
```

Comment:
These definitions are all already specified in `info.json` (`WS2812_PIO_USE_PPIO1` is invalid and does not exist)